index.md

```diff
@@ -9,8 +9,18 @@ so this introduction to Python is built around a common scientific task:
 **data analysis**.
 
 ### Arthritis Inflammation
-We are studying **inflammation in patients** who have been given a new treatment for arthritis, and
-need to analyze the first dozen data sets of their daily inflammation. The data sets are stored in
+We are studying **inflammation in patients** who have been given a new treatment for arthritis. There are 60 patients, who had their inflammation levels recorded for 40 days. We want to analyze these recordings to study the effect of the new arthritis treatment. 
+
+To see how the treatment is affecting the patients in general, we would like to:
+
+1. Calculate the average inflammation per day across all patients.
+2. Plot the result to discuss and share with colleagues.
+
+![](fig/overview.png)
```

Comment:
Regarding the image, the last number in the last row of data, is cut off when I view it. Could the number be removed to prevent that?
![Screenshot_2020-12-12 Programming with Python](https://user-images.githubusercontent.com/6740390/101995156-a7e83380-3c8d-11eb-881f-29c049cf126a.jpg)